common/css/options.css

```diff
@@ -4,8 +4,28 @@
     font-size: 12px; 
     font-weight:normal;
 }
+@media (prefers-color-scheme: dark) {
+	:root {
+		--bg-color: #202023;
+		--color: white;
+	}
+}
+@media (prefers-color-scheme: light) {
+	:root {
+		--bg-color: white;
+		--color: black;
+	}
+}
+
 body {
```

Comment:
Makes the body background & text color match the Firefox Addons page (at least for me).